public_html/wp-content/plugins/wordcamp-speaker-feedback/assets/js/admin.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Update the `dimAfter` callback for feedback replies. This function is called after a comment is unapproved or
+ * approved. In the comments table, it triggers the HTML updates in the admin menu & admin bar (among others).
+ * We can override this by passing in our own function, which only handles the top view + count links.
+ *
+ * See https://core.trac.wordpress.org/browser/trunk/src/js/_enqueues/lib/lists.js?rev=46800#L181
+ * See https://core.trac.wordpress.org/browser/trunk/src/js/_enqueues/lib/lists.js?rev=46800#L593
+ * See https://core.trac.wordpress.org/browser/trunk/src/js/_enqueues/admin/edit-comments.js?rev=47233#L349
+ */
+
+jQuery( document ).ready( function( $ ) {
+	// The `.load()` function can't load into itself, so we need to wrap the views with a container.
+	var $container = $( '<div>' ).html( $( '.subsubsub' ).get( 0 ).outerHTML );
+	$( '.subsubsub' ).replaceWith( $container );
+
+	// We can assume window.theList exists because we're after `edit-comments.js`
+	// And if we know that wpList is an object, we can assume the structure of it is stable.
+	if ( window.theList[0] && 'object' === typeof window.theList[0].wpList ) {
+		window.theList[0].wpList.settings.dimAfter = function() {
+			// Reload the page, but only the top view + count links.
+			$container.load( location.href + ' .subsubsub' );
```

Comment:
TIL that `$.fn.load` can do selective page fragments. 😮

Reply:
Could this also reload a fragment for the comment bubbles in the "In Response To" column?

![comment-bubble](https://user-images.githubusercontent.com/916023/79610642-c588dd00-80ad-11ea-87b2-2c24fe6ea4a8.jpg)

Reply:
hm, this doesn't seem easy to do without doing a request for every part that needs replacing. I tried writing JS for this, but it felt too fragile… So I think a better solution is to just remove the bubble itself, which is what 0496a0fa358821595f39ba1f161334b7e45909fb does. (I also linked to the session's feedback page, rather than the session page).
 
<img width="1191" alt="Screen Shot 2020-04-22 at 3 26 43 PM" src="https://user-images.githubusercontent.com/541093/80025072-0cac0f00-84ae-11ea-98b1-2a9569167352.png">

Reply:
🤔 I guess the bubble's not really necessary, it just makes it match the normal Comments list table a bit better.

Reply:
yeah… but i'm not sure that it's important enough functionality-wise to justify spending more time on it. We can always revisit later.